backend-api/app/ops/users.py

```diff
@@ -3,12 +3,26 @@
 from sqlalchemy.ext.asyncio import AsyncSession
 
 from ..models.users import User
-from ..schemas.wallet import WalletBase
 
 
-async def update_user_wallet(db: AsyncSession, wallet: WalletBase, user: User):
+async def update_user_eth_address(db: AsyncSession, eth_address: str, user: User):
     statement = (
-        sa.update(User).where(User.id == user.id).values(eth_address=wallet.eth_address)
+        sa.update(User).where(User.id == user.id).values(eth_address=eth_address)
     )
     await db.execute(statement)
     await db.commit()
+
+
+async def get_user(db: AsyncSession, user: User):
+    statement = sa.select(User).where(User.id == user.id)
+    result = await db.execute(statement)
+    user = result.scalar_one_or_none()
+    if user is None:
+        return None
+    return user
+
+
+async def delete_user(db: AsyncSession, user: User):
+    statement = sa.delete(User).where(User.id == user.id)
+    await db.execute(statement)
+    await db.commit()
```

Comment:
The app.mangers.UserManager should have this functionality with it already. It's probably safer to use theirs since there might be some other tables that need to be updated on delete.